src/App.js

```diff
@@ -1,9 +1,99 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Form from './components/Form';
+import Todo from './components/Todo';
+import Done from './components/Done';
+import GlobalStyle from './styles/GloalStyle';
+
+const initialTodoData = localStorage.getItem('list')
+  ? JSON.parse(localStorage.getItem('list'))
+  : [];
```

Comment:
이부분은
```suggestion
const initialTodoData = localStorage.getItem('list')??[];
```
혹은
```suggestion
const initialTodoData = localStorage.getItem('list')||[];
```
와 같이 적어도 괜찮을 것 같아요!

Reply:
엄청난걸 배워갑니다 ... 너무깔끔해요

Reply:
허걱스.... 저는 이 코드 쓰고 너무 뿌듯해 하고 있었는데... 이보다 더 간결하게 쓸 수 있었군요.... 충격적이지만 감탄... 감사합니다!!!

Reply:
근데 문기님이 제시해주신 코드에서, 첫번째 코드는 로컬스토리지에 아이템이 존재하는 경우(localStorage.getItem('list')가 true)에는 그 아이템들을 initialTodoData에 할당해줘야 하니까 [ ]가 아니라 JSON.parse(localStorage.getItem('list')가 되어야 할 것 같아요! 그래서 첫번째 코드랑 두번째 코드를 합쳐서 원래 예지님 코드처럼 localStorage.getItem('list') ? JSON.parse(localStorage.getItem('list')) : [ ]로 적는 게 더 간단하지 않을까 하는 저의 작은 의견,,, 그런데 문기님처럼 로지컬 연산자(&&, ||) 활용하는 거 너무 좋은 것 같아요!!

Reply:
앗 JSON.parse부분을 뺐네요 ㅜㅠ 연산자 사용해서 한줄로 굳이굳이 쓰자면 이렇게 쓸 수 있을 것 같습니다
```suggestion
const initialTodoData = JSON.parse(localStorage.getItem('list')||"null")??[];
```